packages/wp-checkout/src/lib/format-monetary-amount-for-locale.js

```diff
@@ -0,0 +1,781 @@
+/* @format */
+
+/**
+ * External dependencies
+ */
+import { isString } from 'lodash';
```

Comment:
I think we can remove this dependency. I'd like to figure out what problems we could encounter with `typeof`.

Reply:
Removed.